Cary_pop/CaryTick

```diff
@@ -0,0 +1,448 @@
+"","Year","Grid","Date","Larvae.m2","Nymphs.m2","Adults.m2","X","day.of.year"
```

Comment:
Do you have permission to make the Cary data public? If not, then these data files should not be in the PR